apps/api/fly.toml

```diff
@@ -0,0 +1,38 @@
+app = "sunodo-api"
+primary_region = "ord"
+
+# [build]
+# builder = "heroku/buildpacks:20"
+# builder = "paketobuildpacks/builder:base"
+# buildpacks = ["gcr.io/paketo-buildpacks/nodejs"]
+
+[[services]]
+internal_port = 8080
+protocol = "tcp"
+
+[[services.ports]]
+handlers = ["http"]
+port = 80
+force_https = true
+
+[[services.ports]]
+handlers = ["tls", "http"]
+port = 443
+
+#[[services.http_checks]]
+#interval = 10000
+#grace_period = "5s"
+#method = "get"
+#path = "/healthz"
+#protocol = "https"
+#restart_limit = 0
+#timeout = 2000
+#tls_skip_verify = false
```

Comment:
do we need this comments?

```suggestion
```

Reply:
Eventually I think we should make this check work, but I couldn't make it work.

Reply:
so we could remove this and open an issue